docs/cpp/Modules.md

```diff
@@ -0,0 +1,108 @@
+
+
+
+
```

Comment:
4 empty lines in the beginning of the file. Was there some top level header that got lost?